### PR TITLE
Add Share button variant

### PR DESCRIPTION
**Pull request by viktoria-mahmud**

### CODE REVIEW

**Review by Cosmin-Daniel — APPROVED** [review]
I've discussed with Viki and this looks good to me.

**Comment by chris-huggins** [comment]
Hi @viktoria-mahmud and @Cosmin-Daniel This looks good to me except for the hover behaviour.

Apologies if there wasn't any direction on this. Hopefully there is already a variable for this, but on hover the background colour of the button should go darker (content remaining black) instead of the content going blue.

I think there should be a colour in use already close to `#D7D9DE` that should make the button body go darker on hover. If it doesn't exist please make a new colour variable along the lines of secondary-button-hover. 

Thanks

**Comment by viktoria-mahmud** [comment]
> Hi @viktoria-mahmud and @Cosmin-Daniel This looks good to me except for the hover behaviour.
> 
> Apologies if there wasn't any direction on this. Hopefully there is already a variable for this, but on hover the background colour of the button should go darker (content remaining black) instead of the content going blue.
> 
> I think there should be a colour in use already close to `#D7D9DE` that should make the button body go darker on hover. If it doesn't exist please make a new colour variable along the lines of secondary-button-hover.
> 
> Thanks

Thanks Chris, the background on hover is added (#d7d9de) and the content color remaining black.

**Review by chris-huggins — APPROVED** [review]

File: assets/sass/patterns/atoms/buttons.scss [hunk 1]
```diff
@@ -116,3 +116,28 @@
...
+  &:hover {
+    background-color: $color-text-ui-background-secondary-grey;
+    border: none;
+    color: $color-text;
+  }
+
+  &.icon-share {
+    @include padding(40, "left");
+    background-image: url("../img/icons/share.svg");
+    background-position: 11px;
```
Comment (discodavey) [inline assets/sass/patterns/atoms/buttons.scss:140]: Can line 140 and 141 be merged into 139 please.

**Pull request merged**